openshift/topics/get_started.adoc

```diff
@@ -155,45 +179,50 @@ Alternatively, perform one of the following procedures to deploy the {project_na
 
 . Log in to the OpenShift web console and select the _sso-app-demo_ project space.
 
-. On the left sidebar, click the `Administrator` tab and then click `</> Developer`.
+. On the left sidebar, click the *Administrator* tab and then click *</> Developer*.
 +
 image:images/choose_developer_role.png[]
 
-. Click `From Catalog`.
+. Click *From Catalog*.
 +
 image:images/add_from_catalog.png[]
 
-. Search for `sso`.
+. Search for *sso*.
 +
 image:images/sso_keyword.png[]
 
-. Choose a template such as `Red Hat Single Sign-On 7.4 on OpenJDK (Ephemeral)`.
+. Choose a template such as *Red Hat Single Sign-On {project_version_base} on OpenJDK (Ephemeral)*.
 +
 image:images/choose_template.png[]
 
-. Click `Instantiate Template`.
+. Click *Instantiate Template*.
 +
 image:images/instantiate_template.png[]
```

Comment:
This image needs to be updated, it has version text on it 7.4

Reply:
Actually, we made an agreement with development that this will be merged into upstream, but not moved to downstream till SSO 7.6.  So the screen will need to change again and I don't have a 7.6 version to use now.